src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/builder/GrantTypeESQueryBuilder.java

```diff
@@ -93,4 +94,17 @@ private void addTermsQueryForClassification(SearchRoleAssignment searchRoleAssig
             parentQuery.must(QueryBuilders.termsQuery(SECURITY_CLASSIFICATION_FIELD_COL, classifications));
         }
     }
+
+    private void addPrefixQueryForCaseAccessCategory(CaseTypeDefinition caseType,
+                                                     SearchRoleAssignment representative,
+                                                     BoolQueryBuilder parentQuery) {
+
+        List<String> caseAccessCategories = getCaseAccessCategories(representative.getRoleAssignment(), caseType);
```

Comment:
Perhaps after this line we should check if `caseAccessCategories` is empty, and if so `return`? Otherwise L#108 will end up adding lots of empty clauses whenever there are no categories.